README.md

```diff
@@ -368,7 +368,7 @@ The package also provides several utility functions.
 
 Parses and validates a HTTP proxy URL. If the proxy requires authentication,
 then the function starts an open local proxy server that forwards to the proxy.
-The port is chosen randomly.
+The port (on which the local proxy server will start) can be set via the `port` property of the first argument, if not provided, it will be chosen randomly.
```

Comment:
No, if the old local proxy server is running, you'd get e.g. `Error: listen EADDRINUSE: address already in use`, because the port is already in use. I believe this is a standard error, it's very similar to cases like trying to start local web servers on the same port.

Reply:
I see, thanks